code/game/objects/items/weapons/misc_items.dm

```diff
@@ -51,15 +51,55 @@
 	w_class = WEIGHT_CLASS_BULKY
 	materials = list(MAT_METAL = 500)
 	attack_verb = list("bludgeoned", "whacked", "cracked")
+	/// Is the secret compartment open?
+	var/is_open = FALSE
+	/// Tiny item that can be hidden on crutches with a screwdriver
+	var/obj/item/hidden = null
 
 /obj/item/crutches/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/two_handed, force_unwielded = 5, force_wielded = 5, icon_wielded = "crutches1")
 
-/obj/item/crutches/update_icon_state()  //Currently only here to fuck with the on-mob icons.
+/obj/item/crutches/Destroy()
+	if(hidden)
+		hidden.forceMove(get_turf(src))
+		hidden = null
+	return ..()
+
+/obj/item/crutches/update_icon_state() //Currently only here to fuck with the on-mob icons.
 	icon_state = "crutches0"
 	return ..()
 
+/obj/item/crutches/attackby(obj/item/I, mob/user, params)
+	. = ..()
+	if(!is_open)
+		return
+	if(!hidden && !isscrewdriver(I) && I.w_class == WEIGHT_CLASS_TINY)
```

Comment:
Please use tool behavior instead, it should look something like this
```suggestion
	if(!hidden && I.tool_behavior != TOOL_SCREWDRIVER && I.w_class == WEIGHT_CLASS_TINY)
```

Reply:
Added in next commit, thanks.